tests/foreman/api/test_remoteexecution.py

```diff
@@ -214,3 +215,5 @@ def test_negative_time_to_pickup(
         query={'search': f'resource_type = JobInvocation and resource_id = {job["id"]}'}
     )
     assert 'The job was not picked up in time' in result[0].humanized['output']
+    global_ttp.value = default_global_ttp
+    global_ttp.update(['value'])
```

Comment:
@pondrejk This still doesn’t completely resolve the original issue. The test can fail before global ttp value is reset.